src/components/screen-reader-notification/screen-reader-notification.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import {Stylesheet} from 'stylable';
+import {SBStateless} from 'stylable-react-component';
+import styles from './screen-reader-notification.st.css';
+
+export type Stylesheet = Stylesheet;
+export const ScreenReaderNotification = SBStateless(
+    ({children}: {children: React.ReactNode}) => {
+        return (
+            <p
+                role="alert"
+                children={children}
```

Comment:
why pass it as prop and not a regular child element?

Reply:
My personal preferences when it comes to components with "simple" children. A bit better to read all props when you use the short tag